tests/integration_tests/functional/test_balloon.py

```diff
@@ -30,15 +28,23 @@ def get_rss_from_pmap():
         _, output, _ = check_output("pmap -X {}".format(pid))
         return int(output.split("\n")[-2].split()[1], 10)
 
-    first_rss = get_rss_from_pmap()
-    time.sleep(1)
-    second_rss = get_rss_from_pmap()
-    abs_diff = abs(first_rss - second_rss)
-    abs_delta = 100 * abs_diff / first_rss
-    print(
-        f"RSS readings: old: {first_rss} new: {second_rss} abs_diff: {abs_diff} abs_delta: {abs_delta}"
-    )
-    assert abs_delta < percentage_delta or abs_diff < 2**10
+    first_rss = 0
+    second_rss = 0
+    for _ in range(5):
+        first_rss = get_rss_from_pmap()
+        time.sleep(1)
+        second_rss = get_rss_from_pmap()
+        abs_diff = abs(first_rss - second_rss)
+        abs_delta = abs_diff / first_rss * 100
+        print(
+            f"RSS readings: old: {first_rss} new: {second_rss} abs_diff: {abs_diff} abs_delta: {abs_delta}"
+        )
+        if abs_delta < percentage_delta:
```

Comment:
is there a reason the `abs_diff < 2**10` is removed? Looks like `abs_diff` is no longer used apart from being printed.

Reply:
It was an arbitrary number as far as I understand and we have a `percentage_delta` anyway, so no real need to it.
Still print `abs_diff` just for debug purposes.